packages/main/src/Table.ts

```diff
@@ -539,11 +522,32 @@ class Table extends UI5Element {
 		return headers;
 	}
 
+	/**
+	 * Refreshes the popin state of the columns.
+	 * Syncs the popin state of the columns with the popin state of the header cells.
+	 * This is needed when additional rows are manually added and no resize happens.
+	 * @private
+	 */
+	_refreshPopinState() {
+		this.headerRow[0]?.cells.forEach((header, index) => {
+			this.rows.forEach(row => {
+				const cell = row.cells[index];
+				if (cell) {
+					cell._popinHidden = header.popinHidden;
+					if (cell._popin !== header._popin) {
```

Comment:
this if seems unnecessary, if values are same it wont invalidate the component

Reply:
Ack